extensions/mailer/runtime/src/main/java/io/quarkus/mailer/runtime/DkimSignOptionsConfig.java

```diff
@@ -6,88 +6,77 @@
 import java.util.OptionalLong;
 
 import io.quarkus.runtime.annotations.ConfigGroup;
-import io.quarkus.runtime.annotations.ConfigItem;
+import io.smallrye.config.WithDefault;
 
 @ConfigGroup
-public class DkimSignOptionsConfig {
+public interface DkimSignOptionsConfig {
```

Comment:
Maybe nest this in `MailerRuntimeConfig`

Reply:
It's pretty long; I would avoid nesting.